modules/installation-aws-regions-with-no-ami.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-government-region.adoc
+
+[id="installation-aws-regions-with-no-ami_{context}"]
+= AWS regions without a published {op-system} AMI
+
+You can deploy an {product-title} cluster to Amazon Web Services (AWS) regions
+without native support for an {op-system-first} AMI or the AWS SDK. When
+deploying to an unknown region, you must specify the custom endpoints that are
+required to access AWS APIs.
+
+[NOTE]
+====
+Deploying to an AWS Government region or specialized public commercial region
+are common use cases where a region would require a custom {op-system} AMI.
+====
+
+You must define the following AWS service endpoints if you provide an unknown
+region.
+
+- `ec2`
+- `elasticloadbalancing`
+- `iam`
+- `route53`
+- `s3`
+- `tagging`
```

Comment:
@abhinavdahiya @katherinedube since we're only recommending GovCloud for 4.6, should I remove info about service endpoint requirements for hidden regions? Perhaps just leave the `install-config.yaml` param info for `platform.aws.serviceEndpoints` to call out that custom service endpoints are supported and omit all other endpoint-related text?

Reply:
After some discussion, I'll pull this info out for now, since we don't want to promote this just yet.